### DISCUSSION TITLE
transpositeur

### DISCUSSION THREAD

**Discussion by BenClam** [GrandOrgue/grandorgue#1083] [General]
Je voudrais pouvoir transposer par 1/2 ton   -3,-2,-1,0,+1,+2,+3 ... avec des boutons tactiles ou des poussoirs (7 dans ce cas)

Est-il possible de le faire dans l'odf?

Le but est que cela soit très facile et rapide d'accès pour l'organiste.

Merci

**Comment by BenClam** [comment]
On pourrait juste agrandir dans le bandeau de GO la fenetre qui permet de faire +1 et -1 avec de grosses touches  pour être tactile  ?

**Reply by rousseldenis** [reply]
Justement, le panneau qui est intégré en standard dans GrandOrgue ne convient-il pas?
